app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/auth/EnrolmentExtractor.scala

```diff
@@ -37,6 +37,7 @@ trait EnrolmentExtractor {
 
   def enrolledForService(loggedInUser: LoggedInUserWithEnrolments, service: Service): Option[Eori] = service match {
```

Comment:
Maybe we could do this instead?
```
  def enrolledForService(loggedInUser: LoggedInUserWithEnrolments, service: Service): Option[Eori] = 
      identifierFor(service.enrolmentKey, "EORINumber", loggedInUser).map(Eori)
```

And then get rid of `enroledCds` and `enrolledATaR` methods?

Reply:
Thought about it, I'll refactor this part

Reply:
Done